### PR TITLE
js-yaml resolution

### PR DESCRIPTION
**Pull request by jhonny218**
## Description
`Resolutions` is used for cases like we are having where the dependencies don't upgrade the dependency needed to be upgraded. Since those sub-dependencies are locked (`package-lock.json`), we can't modify them manually. However, with `resolutions` we can override it.

This PR will upgrade `js-yaml` sub-dependencies to `^3.13.1`.

### Documentation
[Selective dependency resolutions](https://classic.yarnpkg.com/en/docs/selective-version-resolutions/)

## Github Security Issue
https://github.com/department-of-veterans-affairs/vets-website/network/alert/yarn.lock/js-yaml/open

Screen Shot 2020-03-10 at 1 27 42 PM

## Testing done
Locally

## Acceptance criteria
- [ x] Upgrade `js-yaml` to version `3.13.1` or later to fix vulnerability

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -9151,7 +9144,7 @@ lodash.zip@^4.2.0:
   resolved "https://registry.yarnpkg.com/lodash.zip/-/lodash.zip-4.2.0.tgz#ec6662e4896408ed4ab6c542a3990b72cc080020"
   integrity sha1-7GZi5IlkCO1KtsVCo5kLcswIACA=
 
-lodash@4.17.15, "lodash@>=3.5 <5", lodash@^2.4.1, lodash@^4.0.0, lodash@^4.13.1, lodash@^4.15.0, lodash@^4.17.10, lodash@^4.17.11, lodash@^4.17.12, lodash@^4.17.13, lodash@^4.17.14, lodash@^4.17.15, lodash@^4.17.2, lodash@^4.17.4, lodash@^4.2.0, lodash@^4.3.0, lodash@^4.6.1, lodash@~4.16.4:
+"lodash@>=3.5 <5", lodash@^2.4.1, lodash@^4.0.0, lodash@^4.13.1, lodash@^4.15.0, lodash@^4.17.10, lodash@^4.17.11, lodash@^4.17.12, lodash@^4.17.13, lodash@^4.17.14, lodash@^4.17.15, lodash@^4.17.2, lodash@^4.17.4, lodash@^4.2.0, lodash@^4.3.0, lodash@^4.6.1, lodash@~4.16.4:
```
Comment (bkjohnson) [inline yarn.lock]: Do we know why the lodash change was removed?

Comment (U-DON) [reply yarn.lock]: It looks like the `js-yaml` resolution was placed after the `lodash` resolution. Might this be a result of that ordering?

Comment (jhonnyoddball) [reply yarn.lock]: I'm not entirely sure. However, it's the same version as it should be. Maybe because of the `^`

Comment (jhonnyoddball) [reply yarn.lock]: I placed it before `lodash` and it didn't make a difference

**Review by bkjohnson — COMMENTED** [review]
Do we need to be concerned about this?

image

**Comment by jhonnyoddball** [comment]
I don't think that's a problem. I read that it will always show these warnings when they are very old dependencies.

**Review by bkjohnson — APPROVED** [review]

**Pull request merged**